config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatasets.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -92,46 +99,55 @@ spec:
                       - targetTypes
                       type: object
                     domain:
-                      description: |-
-                        A domain to grant access to. Any users signed in with the
-                        domain specified will be granted the specified access.
+                      description: '[Pick one] A domain to grant access to. Any users
+                        signed in with the domain specified will be granted the specified
+                        access. Example: "example.com". Maps to IAM policy member
+                        "domain:DOMAIN".'
                       type: string
                     groupByEmail:
-                      description: An email address of a Google Group to grant access
-                        to.
+                      description: '[Pick one] An email address of a Google Group
+                        to grant access to. Maps to IAM policy member "group:GROUP".'
                       type: string
                     iamMember:
-                      description: |-
-                        Some other type of member that appears in the IAM Policy but isn't a user,
-                        group, domain, or special group. For example: 'allUsers'.
+                      description: '[Pick one] Some other type of member that appears
```

Comment:
Do you happen to know where this `[Pick one]` comes from? Very interesting tag.

Reply:
This is from this proto https://github.com/googleapis/googleapis/blob/master/google/cloud/bigquery/v2/dataset.proto#L172